config/dev.exs

```diff
@@ -98,4 +98,11 @@ config :stripity_stripe,
   api_key: "sk_test_thisisaboguskey",
   api_base_url: "http://localhost:12111/v1/"
 
+config :logflare, Logflare.Vault,
+  ciphers: [
+    default:
+      {Cloak.Ciphers.AES.GCM,
+       tag: "AES.GCM.V1", key: "Sy1NN20+lvHCnE6t6JV5MSp+q1SCrOH82dCOdfwszIo=" |> Base.decode64!()}
```

Comment:
we can shift this up to config.exs, then we don't need to duplicate the config defining for dev and test

Reply:
I did it this way to be less error prone. If by mistake we deploy staging or production without this configured the default value will be used and lead to potential breaches or badly encrypted data.